docs/tables/aws_vpc_flow_log/index.md

```diff
@@ -7,7 +7,18 @@ description: "AWS VPC flow logs capture information about IP traffic going to an
 
 The `aws_vpc_flow_log` table allows you to query data from [AWS VPC Flow Logs](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html). This table provides detailed insights into network traffic within your VPC, including source and destination IP addresses, ports, protocols, and more.
 
-**Note**: For timestamp information, the `start` field will be used first, with the `end` field as a fallback. If neither field is available, then that log line will not be collected and Tailpipe will return an error.
+**Note**:
+- Using `aws_s3_bucket` source:
```

Comment:
I think most of these notes could be explained through examples in the Example Configurations section and aren't required here, it's hard to understand without seeing the configs anyway

Reply:
Removed the notes that are specific to example configuration.